metrics/src/metrics.rs

```diff
@@ -12,6 +12,46 @@ use std::thread;
 use std::time::{Duration, Instant};
 use sys_info::hostname;
 
+#[macro_export]
+macro_rules! integer {
```

Comment:
Can’t we do some more fancy macro stuff such that the integer! and string!  macros are only available within the submit! macro?  Otherwise those names are super generic and are likely to footgun somebody

Reply:
Yup, the names were placeholders since I'm naming challenged :-)